.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: Build and Test (${{ matrix.os }} / OpenJDK ${{ matrix.jdk }})
+    strategy:
+      fail-fast: true
```

Comment:
fail fast.... does this mean that if the Java8 build fails, java11 is not started anymore?

Reply:
The first build that fails causes the rest to be cancelled or not started. The idea is that it saves build-minutes

Reply:
Ok, but it would make comparing eg Java8 and Java11 results a bit more right? What if Java8 is built first, which fails whilst Java11 is just OK...

Reply:
Then the build will fail. It fails on the first failure. It's a trade-off between saying "any fail is a fail" and I want it to cost less to use CI, or I am willing to pay more and I want to see all results.